drivers/cpl/nuopc/glc_import_export.F90

```diff
@@ -200,26 +212,27 @@ subroutine advertise_fields(gcomp, num_icesheets_in, rc)
     ! Advertise import fields
     !--------------------------------
 
-    ! Note that we advertise the import fields even if running with a non-evolving ice
-    ! sheet; this is needed for the MED -> GLC mapping to work (which we do even for a
-    ! non-evolving ice sheet).
-    call fldlist_add(fldsToGlc_num, fldsToGlc, trim(flds_scalar_name))
-    call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_tsrf)
-    call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_qice)
-
-    ! Now advertise import fields
-    do ns = 1,num_icesheets
-       do nf = 1,fldsToGlc_num
-          call NUOPC_Advertise(NStateImp(ns), standardName=fldsToGlc(nf)%stdname, &
-               TransferOfferGeomObject='will provide', rc=rc)
-          if (chkErr(rc,__LINE__,u_FILE_u)) return
-          if (my_task == master_task) then
+    if (cism_evolve) then
+       call fldlist_add(fldsToGlc_num, fldsToGlc, trim(flds_scalar_name))
+       call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_tsrf)
+       call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_qice)
+       call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_so_t_depth, ungridded_lbound=1, ungridded_ubound=nlev_import)
+       call fldlist_add(fldsToGlc_num, fldsToGlc, field_in_so_s_depth, ungridded_lbound=1, ungridded_ubound=nlev_import)
```

Comment:
OK, I think it would make a lot of sense for these ocean-depth fields to be inside of an only "cism evolving" check. But the 2D temperature and surface mass balance should be advertised for both cases.